examples/python/open_photogrammetry_format/main.py

```diff
@@ -0,0 +1,229 @@
+        """
+        Download the dataset if necessary and return the project file.
+
+        Parameters
+        ----------
+        dataset : str
+            Name of the dataset to download.
+        log_as_frames : bool, optional
+            Whether to log the cameras as individual frames, by default True
+        """
+        spec = DATASETS[dataset]
+        if not (DATASET_DIR / spec.dir_name).exists():
+            zip_file = DATASET_DIR / f"{dataset}.zip"
+            if not zip_file.exists():
+                download_file(DATASETS[dataset].url, zip_file)
+            unzip_dir(DATASET_DIR / f"{dataset}.zip", DATASET_DIR)
+
+        return cls(DATASET_DIR / spec.dir_name / "project.opf", log_as_frames=log_as_frames)
+
+    def log_point_cloud(self) -> None:
+        """Log the project's point cloud."""
+        pcl = self.project.point_cloud_objs[0]
+        rr.log_points("world/pcl", positions=pcl.nodes[0].position, colors=pcl.nodes[0].color, timeless=True)
+
+    def log_calibrated_cameras(self) -> None:
+        """
+        Log the project's calibrated cameras as individual frames.
+
+        Logging all cameras in a single frame is also possible, but clutter the default view with too many image views.
+        """
+        sensor_map = {sensor.id: sensor for sensor in self.project.input_cameras.sensors}
+        calib_sensor_map = {sensor.id: sensor for sensor in self.project.calibration.calibrated_cameras.sensors}
+
+        for i, (camera, calib_camera) in enumerate(
+            zip(
+                self.project.camera_list.cameras,
+                self.project.calibration.calibrated_cameras.cameras,
+            )
+        ):
+            if not str(camera.uri).endswith(".jpg"):
+                continue
+
+            if self.log_as_frames:
+                rr.set_time_sequence("image", i)
+                entity = "world/cameras"
+            else:
+                entity = f"world/cameras/{i}"
+
+            sensor = sensor_map[calib_camera.sensor_id]
+            calib_sensor = calib_sensor_map[calib_camera.sensor_id]
+
+            omega, phi, kappa = tuple(np.deg2rad(a) for a in calib_camera.orientation_deg)
+            rot = (
+                np.array(
+                    [
+                        [1, 0, 0],
+                        [0, np.cos(omega), -np.sin(omega)],
+                        [0, np.sin(omega), np.cos(omega)],
+                    ]
+                )
+                @ np.array(
+                    [
+                        [np.cos(phi), 0, np.sin(phi)],
+                        [0, 1, 0],
+                        [-np.sin(phi), 0, np.cos(phi)],
+                    ]
+                )
+                @ np.array(
+                    [
+                        [np.cos(kappa), -np.sin(kappa), 0],
+                        [np.sin(kappa), np.cos(kappa), 0],
+                        [0, 0, 1],
+                    ]
+                )
```

Comment:
What is this interpretation of Eulerangles based on? There is something like 12 different ways Euler angles can be interpreted (https://en.wikipedia.org/wiki/Euler_angles) so I'm wondering if the permutation matrix below comes from this code interpreting the Euler angles differently than intended. That is: applying the omega/phi/kappa in a different order might be the solution.